en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md

```diff
@@ -332,7 +412,23 @@ This section involves setting up the API Publisher node and enabling it to work
         url = "https://[devportal-hostname]:${mgt.transport.https.port}/devportal"
         ```
 
-4.  If you need to configure High Availability (HA) for the Api Publisher nodes, use a copy of the active instance configured above as the second active Publisher instance and configure a load balancer fronting the two Publisher instances.
+    4.  Configure the Publisher with the Key Manager.
+    
+    ``` toml tab="Key Manager with HA"
+        [apim.key_manager]
+        service_url = "https://[Key-Manager-LB-host]/services/"
+        username = "$ref{super_admin.username}"
+        password = "$ref{super_admin.password}"
+    ```
+        
+    ``` toml tab="Single Key Manager"
+        [apim.key_manager]
+        service_url = "https://[Key-Manager-host]:${mgt.transport.https.port}/services/"
+        username = "$ref{super_admin.username}"
+        password = "$ref{super_admin.password}"
+    ```
+
+2.  If you need to configure High Availability (HA) for the Api Publisher nodes, use a copy of the active instance configured above as the second active Publisher instance and configure a load balancer fronting the two Publisher instances.
```

Comment:
```suggestion
2.  If you need to configure High Availability (HA) for the API Publisher nodes, use a copy of the instance configured above as the second active Publisher instance and configure a load balancer fronting the two Publisher instances.
```

Reply:
Shall we rephrase is as " If you need to configure High Availability (HA) for the API Publisher nodes, use a copy of the instance configured above as the second active Publisher instance. Thereafter, configure a load balancer fronting the two Publisher instances."